Makefile

```diff
@@ -52,6 +52,11 @@ test: $(TEST_BACKSTOP) $(DATA_FILES)
 
 check: test
 
+# Comprehensive regression test
+.PHONY: dark_regress
+dark_regress: $(TEST_BACKSTOP) $(DATA_FILES)
```

Comment:
I wasn't planning on adding any of the outputs to the archived git-controlled directory, but it was handy to have a script to just run all the dark cals (and the mock dark cals in some of the testing products).  Outputs look as expected.